patches/0014-add-a-sysctl-to-enable-disable-tcp_collapse-logic.patch

```diff
@@ -0,0 +1,164 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "mfreemon@cloudflare.com" <mfreemon@cloudflare.com>
+Date: Tue, 1 Mar 2022 17:06:02 -0600
+Subject: [PATCH] Add a sysctl to skip tcp collapse processing when the receive
+ buffer is full.
+
+For context and additional information about this patch, see the
+blog post at
```

Comment:
do we have anything to link here?

Reply:
The public link isn't available yet.  I'll update this when it is.